### PR TITLE
docs(tutorial): add tuto penpot

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
Tutorial Penpot with Docker Instant App

### CODE REVIEW

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+meta:
+  title: How to deploy a self-hosted Penpot Instance using the Docker InstantApp
+  description: This page shows you how to deploy a self-hosted Penpot Instance using the Docker InstantApp
+content:
+  h1: How to ddeploy a self-hosted Penpot Instance using the Docker InstantApp
```
Comment (nedaehm) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:6]:
```suggestion
  h1: How to deploy a self-hosted Penpot Instance using the Docker InstantApp
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  paragraph: This page shows you how to deploy a self-hosted Penpot Instance using the Docker InstantApp
+tags: penpot docker self-hosted
+categories:
+  - compute
+dates:
+  validation: 2022-09-28
+  posted: 2022-09-28
+---
+
+Penpot is the an open-source design and prototyping platform meant for cross-domain teams. It is completely web based and works with open web standards (SVG). Penpot can run in almost any modern browser, no matter of the underlying operating system.
```
Comment (nedaehm) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:16]:
```suggestion
Penpot is an open-source design and prototyping platform meant for cross-domain teams. It is completely web based and works with open web standards (SVG). Penpot can run in almost any modern browser, no matter of the underlying operating system.
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+In this tutorial you will learn how to deploy your own Penpot Instance using the Docker [InstantApp](https://www.scaleway.com/en/docs/tutorials/docker-instantapp/).
+
+## Installing Penpot using Docker Compose 
+
+1. In the Scaleway console click **Instances** in the **Compute** section of the navigation bar. Then [deploy](https://console.scaleway.com/instance/servers/create?area=fr-par-1&imageKey=c1b530d8-0ca0-45c4-80db-ba06608287b2&offerName=PLAY2-MICRO) a new Instance using the Docker InstantApp].
```
Comment (nedaehm) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:22]: Il y a un crochet en trop après InstantApp, or did you want to put a link maybe?

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+4. Open the file containing the environment variables (`config.env`) in a text editor:
+    ```
+    nano config.env
+    ```
+
+5. Set the `PENPOT_PUBLIC_URI` to your Penpot domain. Then save the file and exit the editor.
+    ```
+    PENPOT_PUBLIC_URI=https://penpot.mydomain.eu
+    ```
+6. Update the APT package cache on your system to make sure to have the latest releases available.
```
Comment (nedaehm) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:45]:
```suggestion
6. Update the APT package cache on your system to make sure you have the latest releases available.
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+            proxy_set_header X-Nginx-Proxy true;
+            proxy_redirect off;
+        }
+    }
+    ```
+10. Link the configuration file to enable it with Nginx.
+    ```
+    ln -s /etc/nginx/sites-available/penpot.conf /etc/nginx/sites-enabled/penpot.conf
+    ```
+11. Reload Nginx to activate the new configuration
```
Comment (nedaehm) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:94]:
```suggestion
11. Reload Nginx to activate the new configuration:
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+```
+
+## Running Penpot 
+
+1. Start Penpot using `docker-compose`:
+    ```
+    docker-compose -p penpot -f docker-compose.yaml up -d
+    ```
+
+2. Add a Penpot user by running the following command.
```
Comment (nedaehm) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:149]:
```suggestion
2. Run the following command to add a Penpot user:
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+Congratulations! You have successfully enabled https://your.domain.com
+
+You should test your configuration at:
+https://www.ssllabs.com/ssltest/analyze.html?d=your.domain.com
+- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+```
+
+## Running Penpot 
+
+1. Start Penpot using `docker-compose`:
```
Comment (nedaehm) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:144]:
```suggestion
1. Run `docker-compose` to start Penpot:
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx (hunk 3 above)
Comment (bene2k1) [reply tutorials/deploy-penpot-with-docker-instantapp/index.mdx:22]:
```suggestion
1. In the Scaleway console click **Instances** in the **Compute** section of the navigation bar. Then [deploy](https://console.scaleway.com/instance/servers/create?area=fr-par-1&imageKey=c1b530d8-0ca0-45c4-80db-ba06608287b2&offerName=PLAY2-MICRO) a new Instance using the Docker InstantApp.
```

**Review by nedaehm — APPROVED** [review]

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+  paragraph: This page shows you how to deploy a self-hosted Penpot Instance using the Docker InstantApp
+tags: penpot docker self-hosted
+categories:
+  - compute
+dates:
+  validation: 2022-09-28
+  posted: 2022-09-28
+---
+
+Penpot is an open-source design and prototyping platform meant for cross-domain teams. It is completely web based and works with open web standards (SVG). Penpot can run in almost any modern browser, no matter of the underlying operating system.
```
Comment (RoRoJ) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:16]:
```suggestion
Penpot is an open-source design and prototyping platform designed for cross-domain teams. It is completely web based and works with open web standards (SVG). Penpot can run in almost any modern browser, regardless of the underlying operating system.
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+
+In this tutorial you learn how to deploy your own Penpot Instance using the Docker [InstantApp](https://www.scaleway.com/en/docs/tutorials/docker-instantapp/).
+
+## Installing Penpot using Docker Compose 
+
+1. In the Scaleway console click **Instances** in the **Compute** section of the navigation bar. Then [deploy](/compute/instances/how-to/create-an-instance/) a new Instance using the [Docker InstantApp](https://console.scaleway.com/instance/servers/create?area=fr-par-1&imageKey=c1b530d8-0ca0-45c4-80db-ba06608287b2&offerName=PLAY2-MICRO).
```
Comment (RoRoJ) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:22]:
```suggestion
1. Click **Instances** in the **Compute** section of the [Scaleway console](https://console.scaleway.com/organization) side menu. Then [deploy](/compute/instances/how-to/create-an-instance/) a new Instance using the [Docker InstantApp](https://console.scaleway.com/instance/servers/create?area=fr-par-1&imageKey=c1b530d8-0ca0-45c4-80db-ba06608287b2&offerName=PLAY2-MICRO).
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+2. Run the following command to add a Penpot user:
+    ```
+    docker exec -ti penpot-penpot-backend-1 ./manage.sh create-profile
+    ```
+
+3. Open a web browser on your local computer and point it to your Penpot domain. The login screen displays.
+
+    <Lightbox src="scaleway-penpot-login.webp" alt="" />
+
+4. Enter using the credentials of the previously created user. The Penpot dashboard displays. You can start designing your assets now.
```
Comment (RoRoJ) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:158]:
```suggestion
4. Log in using the credentials of the previously created user. The Penpot dashboard displays. You can now start designing your assets. 
```

File: tutorials/deploy-penpot-with-docker-instantapp/index.mdx [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+3. Open a web browser on your local computer and point it to your Penpot domain. The login screen displays.
+
+    <Lightbox src="scaleway-penpot-login.webp" alt="" />
+
+4. Enter using the credentials of the previously created user. The Penpot dashboard displays. You can start designing your assets now. 
+
+    <Lightbox src="scaleway-penpot-dashboard.webp" alt="" />
+
+
+Your Penpot Instance is ready now. For a more advanced configation of additional features, refer to the [official documentation](https://help.penpot.app/technical-guide/configuration/).
```
Comment (RoRoJ) [inline tutorials/deploy-penpot-with-docker-instantapp/index.mdx:163]:
```suggestion
Your Penpot Instance is now ready. For a more advanced configuration of additional features, refer to the [official documentation](https://help.penpot.app/technical-guide/configuration/). 
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**